src/error-handling.js

```diff
@@ -20,7 +20,18 @@ async function validatePathName (dir) {
   }
 }
 
-// Each file must have .json extension, and each file must be syntactically correct, and no file is an array of schema
+// If a file holds an array of schema, each array element must be an object-type
+function validateArrayOfSchema (jsonArray, file) {
+  jsonArray.forEach(function (schema, index) {
+    if (!(schema instanceof Object) || Array.isArray(schema)) {
```

Comment:
The `|| Array.isArray(schema)` condition seems to imply that nested arrays are supported. Is that true?

Reply:
No, they are not supported in this version.  If it's anything besides an Object, it should throw an error.  I had to check for an array specifically because `[] instanceof Object` actually returns `true`